src/utils/events.ts

```diff
@@ -0,0 +1,150 @@
+import dappcon22 from '../images/conferences/dappcon22.jpg';
+import blockchaineurope2022 from '../images/conferences/blockchaineurope2022.png';
+import devcon6 from '../images/conferences/devcon6.jpeg';
+import ethdownunder from '../images/conferences/ethdownunder.png';
+import devcon6Icon from '../images/event-icons/devcon6.jpg';
+import { DateTime } from 'luxon';
+import { crowDistance } from './geo';
+
+export type EventItemProps = {
+  name: string;
+  location: string;
+  image: string;
+  url?: string;
+  conferenceUrl?: string;
+  date: string;
+  latlon?: [number, number];
+  mapIcon?: {
+    url: string;
+    rounded?: boolean;
+  };
+  dateRange: {
+    fromDate: string;
+    toDate: string;
+  };
+};
+
+export const upcomingEvents: EventItemProps[] = [
```

Comment:
I propose to move such definition to the ./src/config.ts because this array looks like configuration